### ISSUE TITLE
Add Neil Larson's works

### ISSUE THREAD

**Issue by nichtich** [nichtich/hypertext-timeline#39]
Neil Larson seems to have been way ahead of his time. He produced several hypertext systems for DOS but lacked commercial marketing so the products were only known to a small audience.

## [MaxThink](http://www.wikidata.org/entity/Q74587961)

<https://awarewriter.wordpress.com/2012/01/29/monday-musings-maxthink-the-only-idea-processor/>

![image](https://user-images.githubusercontent.com/299856/68672963-04d95d80-0553-11ea-8179-16c97f13ab76.png)

## [PC-Hypertext](https://www.wikidata.org/wiki/Q74670748)

![image](https://user-images.githubusercontent.com/299856/68672015-ff7b1380-0550-11ea-9e15-a281099bb676.png)

## [HOUDINI Outline Processor](https://www.wikidata.org/wiki/Q74667091)

![image](https://user-images.githubusercontent.com/299856/68670608-9c3bb200-054d-11ea-9c5a-bef865ce4d7c.png)

image from Encyclopedia of Library and Information Science: Volume 49 - Supplement 12, page 203

## [TransText](https://www.wikidata.org/wiki/Q74679064)

![image](https://user-images.githubusercontent.com/299856/68675933-3bb27200-0559-11ea-97d9-8089fb1f26e3.png)

Screenshot from PC Mag May 1991

## HyperBBS and HyperLAN

Was ist just the BBS or did it provide wiki-like collaborative editing?

**Comment by buckeye43210** [comment]
Don't forget about Perm.

**Comment by nichtich** [comment]
@buckeye43210 do you have any references? The [MaxThink 1994 manual](https://awarewriter.files.wordpress.com/2014/03/maxthink.pdf) only mentions PERM in one sentence but it is not specific about PERM:

> But if your MaxThink outline or the MaxThink file created by our PERM expert system has 1,000 to 10,000 or more subtopics, this link-naming approach is the only way to fly.

So PERM seems to have been an expert system and its content was exported to MaxThink?

**Comment by buckeye43210** [comment]
Yes, and the perm output file can be converted to a mind map via tools like text-to-freemind.
